src/frequenz/sdk/microgrid/client/_client.py

```diff
@@ -212,7 +212,7 @@ async def components(self) -> Iterable[Component]:
         try:
             component_list = await self.api.ListComponents(
                 microgrid_pb.ComponentFilter(),
-                timeout=DEFAULT_GRPC_CALL_TIMEOUT,
+                timeout=int(DEFAULT_GRPC_CALL_TIMEOUT),
```

Comment:
I don't remember exactly, but it was very likely because I was hinted by `mypy`. Maybe it also gets the type info wrong as with the `request_iterator`... :thinking: 

I will try to revert it and see what's complaining about and add a `type: ignore` if it's wrong.

Thanks for checking!

Reply:
```
src/frequenz/sdk/microgrid/client/_client.py:227: error: Argument "timeout" to "__call__" of "UnaryUnaryMultiCallable" has incompatible type "float"; expected "Optional[int]"  [arg-type]
```

I'll add an ignore :+1: